WordPress/Classes/ViewRelated/Reader/Reader.storyboard

```diff
@@ -346,8 +346,19 @@
                                                 </connections>
                                             </button>
                                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="yq2-Pz-GyO">
-                                                <rect key="frame" x="48" y="0.0" width="192" height="50"/>
+                                                <rect key="frame" x="48" y="0.0" width="136" height="50"/>
                                             </view>
+                                            <button opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" horizontalCompressionResistancePriority="1000" contentHorizontalAlignment="right" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="piB-Zd-RVQ" customClass="PostMetaButton">
+                                                <rect key="frame" x="200" y="13" width="40" height="24"/>
+                                                <constraints>
+                                                    <constraint firstAttribute="width" relation="greaterThanOrEqual" constant="30" id="Mum-BO-4Xd"/>
```

Comment:
I noticed the other button width constraints have a priority of 750. I didn't see any spacing issues, but I suspect there was a reason. Maybe the reblog button constraint should follow suit?

<img width="194" alt="button_constraints" src="https://user-images.githubusercontent.com/1816888/70471689-efd8e700-1a8a-11ea-8669-8a217c8daf49.png">

Reply:
Yes, it should be set as the others